### ISSUE TITLE
Add keyboard shortcut to show keyboard shortcuts to top of list

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#16691]
We show the keyboard shortcut command on the menu before the actual modal showing them. We then show it but it's hard to find in a sea of options.

![image](https://user-images.githubusercontent.com/253067/61562723-d696b600-aa69-11e9-9bd6-c73919ccadbc.png)

It could be great to like Slack show on the actual section:

![image](https://user-images.githubusercontent.com/253067/61562703-c979c700-aa69-11e9-9301-1376dfe970fe.png)

**Comment by karmatosed** [comment]
Here is what I'd suggest which moves the hidden shortcut to a more prominent position:

![Artboard](https://user-images.githubusercontent.com/253067/61653757-ed7c1900-acb2-11e9-95e0-1c664c8caa01.png)

**Comment by mapk** [comment]
[Today's design meeting in slack](https://wordpress.slack.com/archives/C02S78ZAL/p1563900517071700) resulted in an agreed decision to move the shortcut to the top in its own section. It feels better and brings proper hierarchy to this action.
